docs/source/release_notes/release-notes.rst

```diff
@@ -98,132 +106,157 @@ develop
-         - Timelock server no longer appends client names to metrics. Instead, each metric is aggregated across all clients.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2501>`__)
-
-    *    - |improved|
-         - Timelock server will now gain leadership synchronously, if possible, the first time a new client namespace is requested. Previously, the first request would always return 503.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2503>`__)
-
     *    - |improved|
-         - We now report metrics for Transaction conflicts.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2540>`__)
-
-    *    - |improved|
-         - ``getRange`` is now more efficient when scanning over rows with many updates in Cassandra, if just a single column is requested.
+         - ``getRange`` is now more efficient when scanning over rows with many updates in Cassandra if just a single column is requested.
            Previously, a range request in Cassandra would always retrieve all columns and all historical versions of each column, regardless of which columns were requested.
            Now, we only request the latest version of the specific column requested, if only one column is requested. Requesting multiple columns still results in the previous behavior, however this will also be optimized in a future release.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2480>`__)
 
-    *    - |fixed|
-         - ``SerializableErrorDecoder`` will decode error properly instead of throwing ``NullPointerException``.
-
     *    - |deprecated| |improved|
          - ``SerializableTransactionManager`` is now created via an immutable builder instead of a long list of individual arguments. Use ``TransactionManagers.builder()``
            to get the builder and once completely configured, build the transaction manager via the builder's ``.buildSerializable()`` method.
            The existing ``create`` methods are deprecated and will be removed by November 15th, 2017.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2459>`__)
 
-    *    - |fixed|
-         - Async Initialization now works with TimeLock Server.
-           Previously, for Cassandra we would attempt to immediately migrate the timestamp bound from Cassandra to TimeLock on startup, which would fail if either of them was unavailable.
-           For DBKVS or other key-value services, we would attempt to ping TimeLock on startup, which would fail if TimeLock was unavailable (though the KVS need not be available).
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2518>`__)
+    *    - |devbreak|
+         - ``TransactionManagers.builder()`` no longer has a ``callingClass(..)`` method and now requires the consumer to directly specify their user agent via the previously optional method ``userAgent(..)``. All of the ``TransactionManagers.create(..)`` methods are still deprecated.
```

Comment:
nit: this one doesn't seem like an improvement.

I'm not sure where to put this, though. Perhaps we could reword this to say that this is part of efforts to ensure that users explicitly provide user agents in a consistent, streamlined way.